lib/sendf.c

```diff
@@ -237,7 +237,9 @@ void Curl_infof(struct Curl_easy *data, const char *fmt, ...)
     size_t len;
     char print_buffer[2048 + 1];
     va_start(ap, fmt);
-    vsnprintf(print_buffer, sizeof(print_buffer), fmt, ap);
+    len = vsnprintf(print_buffer, sizeof(print_buffer), fmt, ap);
+    if(len >= sizeof(print_buffer))
+      snprintf(print_buffer + (sizeof(print_buffer) - 5), 5, "...\n");
```

Comment:
I'm not a fan of magic numbers, but the alternatives didn't seem terribly appealing either. Any better suggestions?

Reply:
Not a fan either, but in this case the plain number seems like the better choice.